src/replication/replicator.rst

```diff
@@ -127,8 +127,11 @@ The replication job will also appear in
       }
 
 ``_scheduler/jobs`` shows more information such as a detailed history of
-state changes. However if replication has completed or has failed to
-start it would not appear here, only in ``_scheduler/docs``.
+state changes. If a replication backed by a document has not yet started,
```

Comment:
Can we use consistent terminology, and call this a "persistent replication" here, too?

```suggestion
state changes. If a persistent replication has not yet started,
```

Reply:
I was afraid of "introducing" a new terminology.